docs/ism/policies.md

```diff
@@ -60,24 +60,27 @@ This table lists the parameters that you can define for an action.
 
 Parameter | Description | Type | Required | Default
 :--- | :--- |:--- |:--- |
-`timeout` |  The timeout period for the action. Accepts time units for minutes, hours, and days. | `time unit` | No | Specific to action
+`timeout` |  The timeout period for the action. Accepts time units for minutes, hours, and days. | `time unit` | No | -
 `retry` | The retry configuration for the action. | `object` | No | Specific to action
-`count` | The number of retry counts. | `number` | No | Specific to action
-`backoff` | The backoff policy type to use when retrying. | `string` | No | Exponential
-`delay` | The time to wait between retries. Accepts time units for minutes, hours, and days. | `time unit` | No | Specific to action
 
-All actions have common `timeout`, `retry`, and `backoff` settings. If you don't specify these settings, ISM uses its defaults.
+The `retry` operation has the following parameters:
```

Comment:
Are we comfortable calling these parameters rather than fields?

Reply:
I have used "parameters" when the user can specify them as inputs and "fields" when some of them are read only and used as a reference. Is that ok?

Reply:
Yep, that's fine. Just wanted to make sure there was a reason and it wasn't an oversight.